openbas-front/src/admin/components/simulations/simulation/ExercisePopover.tsx

```diff
@@ -199,6 +206,14 @@ const ExercisePopover: FunctionComponent<ExercisePopoverProps> = ({
           )}
         </>
       </Drawer>
+      <Drawer
+        open={openReports}
+        containerStyle={{ padding: '0px' }}
```

Comment:
I don't understand why you need this ?

Reply:
to erase the initial padding given into the drawer openbas-front/src/components/common/Drawer.tsx component